src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment: { name, email, body } }) => (
+  <>
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
+        {' by '}
+
+        <a className="CommentInfo__email" href={`mailto:${email}`}>
+          {email}
+        </a>
+      </div>
+
+      <div className="CommentInfo__body">{body}</div>
+    </div>
+  </>
```

Comment:
Here is an extra wrapper for the red component, since we have only one element.
```suggestion
    <div className="CommentInfo">
      <div className="CommentInfo__title">
        <strong className="CommentInfo__name">{name}</strong>

        {' by '}

        <a className="CommentInfo__email" href={`mailto:${email}`}>
          {email}
        </a>
      </div>

      <div className="CommentInfo__body">{body}</div>
    </div>
```